### PR TITLE
BarcelonaJS Job Site

### PR DESCRIPTION
**Pull request by PatrickHeneise**
this is a simple setup to convert the issue list into a job site for http://jobs.barcelonajs.org. There's no styling yet or anything, as there's no job on the list yet. Would love to get the issue template in and then try this out, extending the site as needed.

Preview: http://patrickheneise.me/jobs/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      mounted: function () {
+        this.getJobs()
+      },
+      methods: {
+        getJobs: function () {
+          var that = this
+          var converter = new showdown.Converter()
+          converter.setFlavor('github');
+
+          axios.get('https://api.github.com/repos/patrickheneise/jobs/issues?labels=jobs')
```
Comment (Gpx) [inline index.html]: Is this the correct URL?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            .then(function (res) {
+              var jobs = res.data.map(function (job) {
+                console.log(showdown.getDefaultOptions())
```
Comment (Gpx) [inline index.html]: Do we need to log this?

File: index.html (hunk 1 above)
Comment (PatrickHeneise) [inline index.html]: Labels etc. need to be set up on the repo to properly test this, right now it's linking to my test repo.

File: index.html (hunk 2 above)
Comment (PatrickHeneise) [inline index.html]: Oops.

**Pull request closed without merging**